src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -112,6 +121,11 @@ def create_llm_inputs(
         add_stream:
             If true adds a steam field to each payload
 
+        Required Synthetic Prompt Generation Parameters
```

Comment:
this will be required for all modes no?
Won't URL and file also need this to generate input token metrics?
Do you envision doing that here and passing the values to the metrics or passing the inputs along and letting the metrics module handle that work?

Reply:
It depends on how we choose to implement the metrics calculations (either inside this class or in another).

Reply:
Longer term I expect that the input token calculation (for outputting statistics) will be done elsewhere, and this would only be needed for synthetic generation